### ISSUE TITLE
Process hangs in 0.13.0

### ISSUE THREAD

**Issue by TimvdEijnden** [szwacz/fs-jetpack#46]
When using fs-jetpack 0.13.0 in an electron app I get the following error:
```
(node:3415) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: EEXIST: file already exists, mkdir
```

The process hangs and won't continue.
Tried with Node v6.10.0 & v7.6.0.

The problem started after an upgrade from 0.12.0 to 0.13.0. After a downgrade back to 0.12.0 it started to work again.

**Comment by szwacz** [comment]
Thanks for reporting. 
- What operating system are you on?
- Were you able to track down the line in your code which triggers the error?

**Comment by TimvdEijnden** [comment]
I'm on macOS Sierra 10.12.3 (16D32).

The following code triggered the error:
```js
return projectDir.copyAsync('app', destDir.path(), {
    overwrite: true,
    matching:[
        './node_modules/**',
         './helpers/**',
        './**/*.html',
        './**/*.+(jpg|png|svg|gif)',
         './**/*.js'
        ]
    });
```

**Comment by szwacz** [comment]
Indeed found one case where promise rejection wasn't handeled, and fixed it in 0.13.1. @TimvdEijnden could you please test with that version? I have no idea what will happen :)

**Comment by TimvdEijnden** [comment]
I've just tested with 1.13.1 While the error is fixed my script still hangs because of this error:
```
Error: EEXIST: file already exists, mkdir '/Users/tim/git/xxx/build/javascripts/helpers'
    at Error (native)
```

in  1.13.0 the error is silent and the script will continue to run. So I can fix this my modifying my script.

**Comment by szwacz** [comment]
I'm trying to fix it a bit blindly, but hoping version 0.13.2 is free of this issue. Please confirm :)

**Comment by TimvdEijnden** [comment]
Yes you're right! It works 👍
